src/activities/assignments/AssignmentEntity.js

```diff
@@ -610,6 +614,22 @@ export class AssignmentEntity extends Entity {
 			fields.push({ name: 'completionType', value: assignment.completionType });
 		}
 
+		if (typeof assignment.isIndividualAssignmentType !== 'undefined') {
+			if (assignment.isIndividualAssignmentType !== this.isIndividualAssignmentType()) {
+				fields.push({ name: 'isIndividualAssignmentType', value: assignment.isIndividualAssignmentType });
```

Comment:
This parameter is not present on [the controller](https://search.d2l.dev/xref/lms/le/dropbox/D2L.LE.Dropbox.Extensibility/HM/Folders/Controllers/FoldersController.cs?r=65c56194#203. What are you trying to do here?

_I did not mean to delete the previous version of this comment._

Reply:
Oh good catch! 🤦‍♂ I think we could just remove this if statement? Wouldn't the section below this be sufficient at setting the folder type?

Reply:
I think so, yeah.